src/common/vordr.ts

```diff
@@ -0,0 +1,64 @@
+import { isInSubnet, isIP } from 'is-in-subnet';
+import { Context } from '../Context';
+import { Comment, User } from '../entity';
+import { logger } from '../logger';
+import { counters } from '../telemetry';
+
+const vordrIPs =
+  process.env.VORDR_IPS?.split(',').filter((ip) => Boolean(ip)) || [];
+
+const vordrWords =
+  process.env.VORDR_WORDS?.split(',').filter((word) => Boolean(word)) || [];
+
+export const validateVordrIPs = (ip: string): boolean =>
+  isIP(ip) && isInSubnet(ip, vordrIPs);
+
+export const checkWithVordr = async (
+  comment: Comment,
+  { userId, con, req }: Context,
+): Promise<boolean> => {
+  const user = await con.getRepository(User).findOneByOrFail({ id: userId });
```

Comment:
Let's select only the required fields. User object is quite big already

Reply:
Sounds good, I'll select just the flags

Reply:
Refactored to only select flags